render/animation/sequence.go

```diff
@@ -0,0 +1,51 @@
+package animation
+
+import (
+	"image"
+
+	"github.com/fogleman/gg"
+	"tidbyt.dev/pixlet/render"
+)
+
+// Sequence renders a list of `Animate` widgets in sequence.
```

Comment:
Actually, I believe this widget could live as `render.Sequence` and be a more advanced version of `render.Animation`, supporting animated children.